scanrepository/scanrepository.go

```diff
@@ -124,16 +124,19 @@ func (cfp *ScanRepositoryCmd) scanAndFixBranch(repository *utils.Repository) (er
 func (cfp *ScanRepositoryCmd) setCommandPrerequisites(repository *utils.Repository, client vcsclient.VcsClient) (err error) {
 	// Set the scan details
 	cfp.scanDetails = utils.NewScanDetails(client, &repository.Server, &repository.Git).
-		SetXrayGraphScanParams(repository.Watches, repository.JFrogProjectKey, len(repository.AllowedLicenses) > 0).
+		SetJfrogVersions(cfp.XrayVersion, cfp.XscVersion).
 		SetFailOnInstallationErrors(*repository.FailOnSecurityIssues).
 		SetFixableOnly(repository.FixableOnly).
 		SetConfigProfile(repository.ConfigProfile).
 		SetSkipAutoInstall(repository.SkipAutoInstall).
 		SetAllowPartialResults(repository.AllowPartialResults).
 		SetDisableJas(repository.DisableJas)
 
-	cfp.scanDetails.XrayVersion = cfp.XrayVersion
-	cfp.scanDetails.XscVersion = cfp.XscVersion
+	repositoryInfo, err := client.GetRepositoryInfo(context.Background(), cfp.scanDetails.RepoOwner, cfp.scanDetails.RepoName)
```

Comment:
remove this call since you have the CloneInfo.HTTP fetched few lines later (either from repository.Git.RepositoryCloneUrl or from a new call I made that we must have at this point)